agent/containers/images/Dockerfile.base.j2

```diff
@@ -8,9 +8,10 @@ COPY ./{{ pbench_repo_file }} /etc/yum.repos.d/pbench.repo
 # ... and make sure we have a proper pbench-agent.cfg file in place;
 # ... and finally, ensure the proper pbench-agent environment variables are set up.
 RUN \
-{% if distro_image.startswith('centos') %}
-    {{ pkgmgr }} install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-{{ distro_image.split(':', 1)[1] }}.noarch.rpm{% if distro_image == 'centos:7' %} centos-release-scl-rh centos-release-scl{% endif %} && \
+{% if distro_image == 'centos:8' %}
+    {{ pkgmgr }} module -y enable python36 && \
+    {{ pkgmgr }} module -y disable python38 && \
```

Comment:
Huh. Interesting. My RHEL 8 laptop shows Python36 [d][e] and Python38 [d]. Never even knew about "modules" before.

Reply:
Yeah, this is wacky.